lenmorld/theshop
# MVP push 1

- Setup eslint + prettier in VSCode
- Add deps
- Add form fields and input animations
- ESLint + Prettier fixes
- test and archive code
- Add to cart from Products - CartSummary component - Modal - addToCart - hideCart
- Implement currency and date formats
- Add CSS module styles
- Customize document to add portal for Modal
- Implement cart page with CartDetails component
- Implement contact and billing details with Field component
- Integrate Stripe part 1 - create payment intent - backend: serverless function stripe-create-payment-intent - UI: createPaymentIntent fetch request
- Integreate Stripe part 2 - checkout page - CheckoutForm with Stripe Elements and CustomerFields - confirm payment with Stripe
- Implement order success page where checkoutForm redirects on success
- Add Cart link
- ESLint and cleanup
